.github/workflows/deploy.yml

```diff
@@ -7,4 +7,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Dump event
-        run: jq . $GITHUB_EVENT_PATH
\ No newline at end of file
+        run: jq . $GITHUB_EVENT_PATH
+
+      - name: Make ssh key
+        run: mkdir -p ~/.ssh && touch ~/.ssh/testing.pem && echo "${{secrets.TESTING_PEM}}" > ~/.ssh/testing.pem && chmod 400 ~/.ssh/testing.pem
+          
+      - uses: actions/checkout@v2
```

Comment:
I feel like I've seen examples where all the `uses` lines appear together, at the top of the list. Any official guidance on this so we can keep up with best practices?

Reply:
I don't think there is official guidance but I think it could live at the top. There's benefits to putting them all together or all with their respective tasks. It's up to you as to whether we move it

Reply:
In that case leave it here for now. If we develop a better native feel for this sort of thing, we can always change it up later.